backend/de.metas.business/src/main/java/de/metas/material/maturing/MaturingConfigRepository.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * #%L
+ * de.metas.business
+ * %%
+ * Copyright (C) 2020 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.material.maturing;
+
+import de.metas.organization.OrgId;
+import de.metas.product.ProductId;
+import de.metas.util.Services;
+import lombok.NonNull;
+import org.adempiere.ad.dao.IQueryBL;
+import org.adempiere.model.InterfaceWrapperHelper;
+import org.compiere.model.I_M_Maturing_Configuration_Line;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.adempiere.model.InterfaceWrapperHelper.loadOutOfTrx;
+import static org.adempiere.model.InterfaceWrapperHelper.saveRecord;
+
+@Repository
+public class MaturingConfigRepository
+{
+	private final IQueryBL queryBL = Services.get(IQueryBL.class);
+
+	public MaturingConfigLine getById(@NonNull final MaturingConfigLineId id)
+	{
+		return fromRecord(loadOutOfTrx(id, I_M_Maturing_Configuration_Line.class));
```

Comment:
pls think about caching. 
think on which level we want to cache, i.e.
* shall we cache the all data (header and lines)
* shall we cache per line?
* shall we cache per product?
* .....